src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -179,6 +179,9 @@ function MetadataSidebarRedesign({ api, elementId, fileId, onError, isFeatureEna
         options: PaginationQueryInput,
     ) => metadataTaxonomyFetcher(api, fileId, scope, templateKey, fieldKey, level, options);
 
+    const taxonomyNodeFetcher = async (scope: string, taxonomyKey: string, nodeID: string) =>
```

Comment:
is `async` needed since `await` is not used?

Reply:
Needed since `metadataTaxonomyNodeAncestorsFetcher` returns a promise.